tardis/model/parse_input.py

```diff
@@ -678,6 +678,11 @@ def parse_csvy_radiation_field_state(
             geometry, packet_source
         )
 
+    if np.any(t_radiative < 1000 * u.K):
+        raise ValueError(
+            f"Radiative temperature is too low in shell {np.argmin(t_radiative)} (T_rad = {t_radiative[np.argmin(t_radiative)]})"
```

Comment:
Maybe add a short description of why this is a problem. Like say, "Temperature is too low in shell, temperatures below 1000K (your chosen value) are not accurately handled by Tardis."

Reply:
Something that suggests what the threshold is, and what you need to do to adapt your model so as to avoid a crash.